src/w3d/renderer/texture.h

```diff
@@ -62,6 +62,10 @@ class TextureFilterClass
     void Set_U_Address_Mode(TxtAddrMode mode) { m_uAddressMode = mode; }
     void Set_V_Address_Mode(TxtAddrMode mode) { m_vAddressMode = mode; }
     FilterType Get_Mag_Filter() { return m_magTextureFilter; }
+    FilterType Get_Min_Filter() { return m_minTextureFilter; }
+    FilterType Get_Mip_Mapping() { return m_mipMapFilter; }
+    TxtAddrMode Get_U_Address_Mode() { return m_uAddressMode; }
+    TxtAddrMode Get_V_Address_Mode() { return m_vAddressMode; }
```

Comment:
These Get functions can be `const` safely.

Reply:
fixed.